### PR TITLE
Adjust Code of Conduct page content

### LINKED ISSUE #23: Code of conduct with new design

### PR DESCRIPTION
**Pull request by jeeehaan**
Closes #23

### CODE REVIEW

**Review by milhamm — APPROVED** [review]
LGTM, needs double-check with others
cc @bandungpy/frontend

File: app/routes/coc._index.tsx
Comment (ridhof) [inline app/routes/coc._index.tsx:12]: wondering why is there a commented line here? if its no longer used, let's delete it.

Comment (ridhof) [reply app/routes/coc._index.tsx:12]: design responsiveness all perfect, once its updated, please merge @jeeehaan

Comment (jeeehaan) [reply app/routes/coc._index.tsx:12]: > wondering why is there a commented line here? if its no longer used, let's delete it.

Sorry, I forgot to delete it, will do it soon.

Comment (jeeehaan) [reply app/routes/coc._index.tsx:12]: > design responsiveness all perfect, once its updated, please merge @jeeehaan

Will do, thank you for the review mas.

**Comment by ridhof** [comment]
di-centang aja opsi `Merge without waiting for requirements to be met` terus klik `Merge pull request`
lagi ngebug kayaknya @jeeehaan

**Comment by jeeehaan** [comment]
> di-centang aja opsi `Merge without waiting for requirements to be met` terus klik `Merge pull request` lagi ngebug kayaknya @jeeehaan

Opsi ini ada dimana mas?

**Comment by ridhof** [comment]
oh kalau ga muncul, let me help merging

**Pull request merged**
